snippets/facets.liquid

```diff
@@ -358,6 +358,7 @@
                         {{ filter.label | escape }}
                       </button>
 
+                      {%- assign max_price_amount = filter.range_max | money | strip_html | escape -%}
                       <p class="mobile-facets__info">{{ "products.facets.max_price" | t: price: max_price_amount }}</p>
```

Comment:
I wonder if the assign should then be outside of the conditional so it's declared only once. 🤔 
THoughts ?